src/pcre2_internal.h

```diff
@@ -1908,6 +1909,11 @@ typedef struct {
 #define UCD_SCRIPTX(ch)     UCD_SCRIPTX_PROP(GET_UCD(ch))
 #define UCD_BPROPS(ch)      UCD_BPROPS_PROP(GET_UCD(ch))
 #define UCD_BIDICLASS(ch)   UCD_BIDICLASS_PROP(GET_UCD(ch))
+#define UCD_ANY_I(ch) \
```

Comment:
needs also an EBCDIC version

http://www.kostis.net/charsets/ebc1026.htm

Reply:
Philip, Zoltan, I don't think it does? Turkish casing only makes sense in UTF/UCP modes where we have characters U+0130 and U+0131.

All of the UCD_RECORD macros and data are using numeric constants, not character constants like `'A'`. The UCD macros must always assume that a `uint32_t ch` is a Unicode codepoint, not an EBCDIC one.

Reply:
The question is compilation, not meaning. The code should be able to be compiled in EBCDIC systems.
Unfortunately we only have Ze'ev Atlas, who has access to EBCDIC, and he did not reply to my last mail :(

Reply:
This PR just uses numeric constants like `0x0130` and `0x49`, which are valid C code on any machine.

It will certainly _compile_ on an EBCDIC machine.